src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,6 @@
-const booleanToWord = boolean => {};
+const booleanToWord = boolean => {
+  if (boolean) return "Yes";
```

Comment:
You could use a ternary here as well

Reply:
eg `boolean ? "Yes" : "No"`

Reply:
I tend to use ternaries when there are only two options, but that's just my style.

Reply:
Ugh I keep forgetting about that, it is much nicer!!